config/import-control.xml

```diff
@@ -60,6 +57,7 @@
   </file>
   <file name="ParserUtil">
     <allow class="antlr.Token"/>
+    <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/>
```

Comment:
@strkkk @rnveach Added this line to allow DetailAstImpl being used here.

Reply:
This is only because you removed the global and added a finer restriction to just the class.
This is fine to me.